### PR TITLE
Update docs to show archived information if connector is not in registries

### PR DESCRIPTION
**Pull request by evantahler**
Archived connectors are in the sidebar, but not the registry lists (e.g. `/integrations/`). 

Also, some header updates

Screenshot 2024-02-20 at 4 57 30 PM

### CODE REVIEW

File: docusaurus/src/components/ConnectorRegistry.jsx [hunk 1]
```diff
@@ -42,7 +42,8 @@ export default function ConnectorRegistry({ type }) {
 
   const connectors = registry
     .filter((c) => c.connector_type === type)
-    .filter((c) => c.name_oss);
+    .filter((c) => c.name_oss)
+    .filter((c) => c.supportLevel_oss); // at lease one connector is missing a support level
```
Comment (postamar) [inline docusaurus/src/components/ConnectorRegistry.jsx:46]:
```suggestion
    .filter((c) => c.supportLevel_oss); // at least one connector is missing a support level
```

File: docusaurus/src/components/HeaderDecoration.jsx [hunk 2]
```diff
@@ -67,14 +67,17 @@ export const HeaderDecoration = ({
...
-          <dd>
-            <a href={github_url} target="_blank">
-              {dockerImageTag}
-            </a>
-          </dd>
-        </div>
+        {supportLevel !== "archived" && (
+          <div>
+            <dt>Latest Version</dt>
+
```
Comment (postamar) [inline docusaurus/src/components/HeaderDecoration.jsx:73]:
```suggestion
```

File: docusaurus/src/remark/utils.js [hunk 3]
```diff
@@ -16,19 +16,52 @@ const isDocsPage = (vfile) => {
...
-
-    return registry.find(
-      (r) => r.dockerRepository_oss === dockerRepository
+  const pathParts = vfile.path.split("/");
+  const connectorName = pathParts.pop().split(".")[0];
+  const connectorType = pathParts.pop();
+  const dockerRepository = `airbyte/${connectorType.replace(
+    /s$/,
+    ""
+  )}-${connectorName}`;
```
Comment (postamar) [inline docusaurus/src/remark/utils.js:25]: curious why this expression is multiline

**Review by postamar — APPROVED** [review]
LGTM as far as I can tell.

File: docusaurus/src/remark/utils.js (hunk 3 above)
Comment (evantahler) [reply docusaurus/src/remark/utils.js:25]: 🤷 it's what my linter did

**Pull request merged**